samigo/samigo-app/src/webapp/jsf/author/publishAssessment.jsp

```diff
@@ -47,6 +48,46 @@
 
   <!-- Error publishing assessment -->
   <h:messages globalOnly="true"  styleClass="sak-banner-error" rendered="#{! empty facesContext.maximumSeverity}" layout="table"/>
+  
+  <!-- CREATE TASK -->
+  <h:panelGroup>
+    <div class="row">
+      <h:outputLabel styleClass="col-md-2" value="#{assessmentSettingsMessages.create_task}" for="createTaskAssess"/>
+      <div class="col-md-10">
+        <t:selectOneRadio id="createTaskAssess" value="#{assessmentBean.createTask}" layout="pageDirection">
+          <f:selectItem itemValue="true" itemLabel="#{assessmentSettingsMessages.create_task_on_widget}"/>
+          <f:selectItem itemValue="false" itemLabel="#{assessmentSettingsMessages.not_create_task_on_widget}"/>
+        </t:selectOneRadio>
+      </div>
+    </div>
+  </h:panelGroup>
+  
+  
+  <!-- NOTIFICATION -->
+  <h:panelGroup>
+    <div class="row" style="margin-top:15px; margin-bottom:15px;">
+      <h:outputLabel styleClass="col-md-2" value="#{assessmentSettingsMessages.notification}" for="createTaskAssess" />
+      <div class="col-md-10">
+        <h:panelGrid>
+          <h:selectOneMenu id="number" value="1" onchange="document.forms[0].submit();">
```

Comment:
This is risky. You're submitting the first form on the page. You should use an id.

Reply:
This "onchange" code is not mine. I just placed all these fields on top of the button bar (they were placed in a weird position, in line with the button bar, with no gap between them), so I'm not changing the functionality here... I just changed the layout to make it more logical.

Reply:
I know, but you should change it anyway just to improve it. Those submits are dangerous. Just add an id to the form and use getElementById in that onchange. Up to you, but it's nice to fix these bits that are near to the areas we are working on, as we go.

Reply:
Ok, I'll change it. Thanks.